src/api_manager/config_manager_test.cc

```diff
@@ -157,25 +157,8 @@ const char kRolloutsResponseMultipleServiceConfig[] = R"(
 }
 )";
 
-// Represents a periodic timer created by API Manager's environment.
-class MockPeriodicTimer : public PeriodicTimer {
+class MockTimerApiManagerEnvironment : public MockApiManagerEnvironmentWithLog {
```

Comment:
Just wondering, can the original segfault be captured by this test?

Reply:
No, this test is using Mock Timer impl.   The crash is caused by Ngx Timer Impl.

Reply:
Please verify it before release.